app/controllers/tours_controller.rb

```diff
@@ -6,4 +6,31 @@ def index
   def show
     @tour = Tour.find(params[:id])
   end
+
+  def create
+    unless check_landmark_with_recording
```

Comment:
Change to if/else instead of unless/else -- flip contents as needed

Reply:
I disagree. If we change it to if/else, it goes against the workflow and makes it confusing.